www/common/common-ui-elements.js

```diff
@@ -594,16 +594,19 @@ define([
         let $icon = $button.find('> i');
 
         let attributes = {};
+        let style = {};
         let btnClass = $button.attr('class');
         let btnId = $button.attr('id');
         let btnTitle = $button.attr('title');
+        let btnVisibility = $button.attr('style');
         if (btnClass) { attributes['class'] = btnClass; }
         if (btnId) { attributes['id'] = btnId; }
         if (btnTitle && !attributes.title) { attributes['title'] = btnTitle; }
-
+        if (btnVisibility) { attributes['style'] = btnVisibility; }
         return UIElements.createDropdownEntry({
             tag: 'a',
             attributes: attributes,
+            style: style,
```

Comment:
I'm not sure I understand why we have a "style" element here if we already have `attributes['style']` above. It seems like one will override the other.